example/spec/helpers/granuleUtils.js

```diff
@@ -238,8 +238,6 @@ const waitForGranuleRecordUpdatedInList = async (stackName, granule, additionalQ
       'beginningDateTime',
       'endingDateTime',
       'error',
-      'execution', // TODO remove after CUMULUS-3698
-      'files', // TODO -2714 this should be removed
```

Comment:
I'm unclear on 2714 (ensure vestigial file records are cleaned up) it *seems* like files should be turned back on wit this PR, but maybe I'm misunderstanding

Reply:
Did a bit more digging into this, this was added here:

https://github.com/nasa/cumulus/pull/2454/files#diff-d7252b4e7a25abd4135d5c5009c4bddcd42099174d4204a39463a92e05b1ab2d

Commenting on `// FUTURE: files contain a "type" field in elasticsearch that does not exist in Postgres`

CUMULUS-2714 https://github.com/nasa/cumulus/pull/2539 updated the logic such that PUT/POST would not leave additional file records on update (it was adding the new ones, modifying existing records ,but not removing files that weren't in file object prior), and this update apparently was missed/wasn't added to the JIRA ticket?   😓  

Given that intent, seems entirely reasonable to remove this if tests are passing....but...given we're tidying up that comment and making this edit, given `granule.files` is an array, we should eval if we need to take the sort action in the comment while comparing files to avoid intermittent integration behaviors.

Reply:
ok, I'll look into it

Reply:
note: ensure we're sorting outputs to make sure there aren't random errors from un-sorted outputs

Reply:
this is sorted to be sure it works right, and comments TODOing it are removed

Reply:
Great, thanks for addressing it.